pkgs/by-name/re/resnap/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenvNoCC,
+  writeShellApplication,
+  ffmpeg,
+  feh,
+  imagemagick_light,
+  lz4,
+  fetchFromGitHub,
+}:
+let
+  name = "reSnap";
+  version = "2.5.2";
+
+  src = stdenvNoCC.mkDerivation {
+    inherit name version;
+    src = fetchFromGitHub {
+      owner = "cloudsftp";
+      repo = "reSnap";
+      rev = "v${version}";
+      hash = "sha256-thVyf1gTDPLQVtZKoWL7SGiWI++ICWqmF/Ar57I3WP8=";
+    };
+    installPhase = ''cp ./reSnap.sh $out'';
+    phases = [
+      "unpackPhase"
+      "patchPhase"
+      "installPhase"
+    ];
+    postPatch = ''
+      substituteInPlace ./reSnap.sh --replace-quiet "\$0" "reSnap"
+    '';
+  };
+in
+writeShellApplication {
```

Comment:
Why not copy the code from src to a subdirectory in out and have a proper application with pname, version, etc?